client/services/Hooks.ts

```diff
@@ -99,11 +99,27 @@ function withAuthModal(isLoggedIn: boolean, cb: (...any) => any) {
   };
 }
 
+function pressEsc(handler: (e: KeyboardEvent) => void) {
```

Comment:
In [this comment](https://github.com/FrontEnd-ro/frontend.ro/pull/504#discussion_r777892740) i suggested naming it `useKeyDown`. This is a general convention among React hooks. Let's stick to it :)

Reply:
I also suggested to refactor the function to be generic. What I mean by that, is I should be able to use it for other keys as well, not just Escape. Example:

```
useKeyDown('Escape', doSmthOnEscape);

...

useKeyCode('Space', doSmthOnSpace)
```

Please refactor.